src/ResourceManager/Network/Commands.Network/Models/Cortex/PSVirtualWan.cs

```diff
@@ -27,5 +31,11 @@ public class PSVirtualWan : PSTopLevelResource
 
         [Ps1Xml(Label = "AllowBranchToBranchTraffic", Target = ViewControl.Table)]
         public bool AllowBranchToBranchTraffic { get; set; }
+
+        [JsonIgnore]
+        public string P2SVpnServerConfigurationsText
```

Comment:
same here, this should have the Ps1Xml label instead of the list

Reply:
P2SVpnServerConfigurationsText. It already has Ps1Xml label.